### PR TITLE
CoFest update: add links for ideas and chat; add CoC

### PR DESCRIPTION
**Pull request by chapmanb**
Refresh CoFest page for online event, cleaning up description, adding
links to a cluster and sign up page. Referencing CoC and actionability.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -273,28 +273,29 @@ enableHackathon: true
...
- - {title: "Preparing for CoFest", image: "hackathon-0.png", details: "Add or expand the cluster ideas document (coming)<br><br>Add an introduction and indicate your preferred working clusters (coming)<br><br>Ask questions on the Conference gitter channel"}
+ - {title: "Registration", image: "hackathon-0.png", details: "CollaborationFest is free, for all 4 days. You still need to register (and select either 2 or 4 days)."}
+ - {title: "Preparing for CoFest", image: "hackathon-0.png", details: "<a href='https://docs.google.com/document/d/1UctntMyOMOFlyxgvZPiWqgXz7R3FjPKnuBVMoNxl9Jc/edit?usp=sharing'>Contribute to the cluster ideas document defining community goals for the two days</a><br><br><a href='https://docs.google.com/spreadsheets/d/1AoSztstDDiJphefjnp7hvZByZKavyusIHzQO3-qAC7M/edit?usp=sharing'>Introduce yourself and indicate your preferred working clusters</a><br><br>Ask questions on the <a href='https://bcc.zulipchat.com/'>Bioinformatics Community Conference Zulip Chat</a>"}
```
Comment (heuermh) [inline _config.yml]: two days &rarr; four days

Comment (chapmanb) [reply _config.yml]: Good catch, thanks so much. Tried to generalize the language.

**Review by heuermh — APPROVED** [review]
Looks great, thank you, @chapmanb!

**Comment by chapmanb** [comment]
Sweet, thank you. I don't have merge privileges to make it all live, so please feel free to mash the green button if you do.

**Comment by denis-yuen** [comment]
(FYI, added/invited both of you as maintainers on the repo, haven't had a chance to look in detail)

**Comment by heuermh** [comment]
Thank you, @denis-yuen!

**Pull request merged**